catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -102,6 +102,8 @@
 
     public static final String CACHED = "cached";
 
+    private static final String INTERNAL_LOCAL_RESOURCE = "internal.local-resource";
```

Comment:
I don't see why the search endpoint needs to know the value of the constant. The logic seems to be based on the attributetype.